components/raftstore/src/store/fsm/store.rs

```diff
@@ -812,7 +813,10 @@ impl<EK: KvEngine + 'static, ER: RaftEngine + 'static, T: Transport>
             StoreTick::PdStoreHeartbeat => self.on_pd_store_heartbeat_tick(),
             StoreTick::SnapGc => self.on_snap_mgr_gc(),
             StoreTick::CompactLockCf => self.on_compact_lock_cf(),
-            StoreTick::CompactCheck => self.on_compact_check_tick(),
+            StoreTick::CompactCheck => {
```

Comment:
Why not directly remove this tick directly ?

Reply:
I am adding configs to gc module, so I am removing obsolete configs in raftstore, which requires removing relevant code. So I am deleting dead code now